### DISCUSSION TITLE
Suggestion: sign releases

### DISCUSSION THREAD

**Discussion by sbrl** [meilisearch/product#540] [Feedback & Feature Proposal]
Signing releases on GitHub with GPG + SHA256 would give people the confidence that releases are both officially released by you and their downloads weren't corrupted - either accidentally or maliciously.

The general process for  this would be to generate SHA256 hashes for your release files (e.g. using sha256sum), and then signing that generated list of sha256 hashes with GPG.

**Comment by curquiza** [comment]
Hello @sbrl thanks for the suggestion, I opened an issue in the Meilisearch repository since it concerns only the release on GitHub and not the Meilisearch product itself

thanks for the suggestion! 😄 

This discussion is replaced by https://github.com/meilisearch/meilisearch/issues/2816
